src/eynollah/eynollah.py

```diff
@@ -226,6 +228,7 @@ def __init__(
         self.model_page_dir = dir_models + "/eynollah-page-extraction_20210425"
         self.model_region_dir_p_ens = dir_models + "/eynollah-main-regions-ensembled_20210425"
         self.model_region_dir_p_ens_light = dir_models + "/eynollah-main-regions_20220314"
+        self.model_region_dir_p_ens_light_only_images_extraction = dir_models + "/eynollah-main-regions_20231127_672_org_ens_11_13_16_17_18"
```

Comment:
Is a new model introduced here @vahidrezanezhad?

Reply:
Yes. eynollah-main-regions_20231127_672_org_ens_11_13_16_17_18 is the new model.

Reply:
Thanks for publishing the model on HF. 

I created a new [models_eynollah.tar.gz](https://github.com/qurator-spk/eynollah/releases/download/v0.3.1/models_eynollah.tar.gz) release asset for `v0.3.1` that includes the new model and also 
- updated [`ocrd-tool.json`](https://github.com/qurator-spk/eynollah/pull/132/commits/351e9a897a390cc5978346ae56bd725f021876d9) 
- and [`Makefile`](https://github.com/qurator-spk/eynollah/pull/132/commits/327b446a16cc0d28281d41c96de1062c18293601) 

@michalbubula Can you please 

> 1. Update [README.md](https://github.com/qurator-spk/eynollah/blob/extracting_images_only/README.md) to document the added functionality

Reply:
I updated the Readme and the Wiki.